package.json

```diff
@@ -39,27 +39,27 @@
   },
   "homepage": "https://github.com/homebridge/camera-utils#readme",
   "dependencies": {
-    "execa": "^5.1.1",
+    "execa": "^6.1.0",
```

Comment:
This is a breaking release of execa. I'm not sure we should include this bump here. Everything else should be fine though

Reply:
okay reverted back to 5.1.1